back/apps/core-fca-low/src/controllers/interaction.controller.ts

```diff
@@ -180,11 +180,15 @@ export class InteractionController {
     const csrfToken = this.csrfService.renew();
     this.sessionService.set('Csrf', { csrfToken });
 
+    const error = req.query.error || undefined;
```

Comment:
todo: req.query.error and req.query.email_suggestion should be validated by nest params validation before being injected in the page.

suggestion: 
- we could test that req.query.error is either not defined or set to the value "invalid_email". The error variable could be a boolean value.
- we could test that req.query.email_suggestion is either not defined or a string of less than 255 chars and a valid email address. If the validation fail we could ignore the field rather than throwing an error.